src/main/java/com/example/bankservicedemo/model/Account.java

```diff
@@ -0,0 +1,33 @@
+package com.example.bankservicedemo.model;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import lombok.Data;
+import lombok.ToString;
+
+@Entity(name = "accounts")
+@Data
+public class Account {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Column(name = "account_number", unique = true, nullable = false)
+    private String accountNumber;
+    @Enumerated(EnumType.STRING)
+    @Column(nullable = false)
+    private Currency currency;
+    private double balance;
```

Comment:
```suggestion
    private BigDecimal balance;
```
please use BigDecimal for money fields